cms/static/js/views/utils/create_course_utils.js

```diff
@@ -1,57 +1,18 @@
 /**
  * Provides utilities for validating courses during creation, for both new courses and reruns.
  */
-define(["jquery", "underscore", "gettext", "js/views/utils/view_utils"],
-    function ($, _, gettext, ViewUtils) {
+define(["jquery", "gettext", "js/views/utils/view_utils", "js/views/utils/create_utils_base"],
+    function ($, gettext, ViewUtils, CreateUtilsFactory) {
+        "use strict";
         return function (selectors, classes) {
-            var validateTotalCourseItemsLength, setNewCourseFieldInErr, hasInvalidRequiredFields,
-                createCourse, validateFilledFields, configureHandlers;
+            CreateUtilsFactory.call(this, selectors, classes);
```

Comment:
Sorry to keep circling on this, but I think part of my confusion is I don't understand what these utility files are returning at this point. Should they be returning the result of CreateUtilsFactory.call(this, selectors, classes)?